docs/organizations/changing-your-plan-and-billing.md

```diff
@@ -2,10 +2,12 @@
 
 Each organization on Codacy has a dedicated plan and associated billing. To make changes to the plan and billing of an organization, open your organization **Settings**, page **Plan and billing**.
 
-<!--github-marketplace-start-->
+<!--plan-caveats-start-->
 !!! note
-    **If you're using GitHub Marketplace,** make changes to your billing details or cancel your plan directly on your [GitHub Billing page](https://github.com/settings/billing).
-<!--github-marketplace-end-->
+    -   If you're using GitHub Marketplace, update your billing details or cancel your plan directly on your [GitHub Billing page](https://github.com/settings/billing).
+
+    -   If you cancel your plan or transition between Enterprise plans, the Codacy UI may display old information for some repositories until they are reanalyzed.
```

Comment:
@RaquelHipolito @ajanczak 

Updated with your suggestions. Let me know, thanks!

Reply:
@nicklem should it be more detailed stating something like: reanalysed manually and/or as new commits happen.

Reply:
@nicklem should it be more detailed stating something like: reanalysed manually and/or as new commits happen.

Reply:
@RaquelHipolito

Good idea. I can link to the [relevant page](https://docs.codacy.com/faq/repositories/how-do-i-reanalyze-my-repository/).

```suggestion
    -   If you cancel your plan or transition between Enterprise plans, the Codacy UI may display old information for some repositories until they're [reanalyzed](../faq/repositories/how-do-i-reanalyze-my-repository.md).
```

Reply:
Upon review, I agree that your suggestion covers a use case I wasn't considering.

```suggestion
    -   If you cancel your plan or transition between Enterprise plans, the Codacy UI may display old information for some repositories until they are reanalyzed manually or as new commits happen.
```